pages/opam-using.html

```diff
@@ -6,25 +6,31 @@ <h2>What is opam?</h2>
 
 <p>Opam is the package manager for the OCaml programming language, the language
 in which Coq is implemented.
-Opam 2 is the recommended version, and is assumed below.
+Opam 2.1 is the recommended version, and is assumed below.
 Instructions on
 <a href="https://opam.ocaml.org/doc/Install.html">how to install opam</a>
 itself are available on the opam website.
 The following command displays the version of opam you have installed:</p>
 
-<pre><code># Make sure opam version is 2.0.0 or above.
+<pre><code># Make sure opam version is 2.1.0 or above.
 opam --version
 </code></pre>
 
 <p>Follow the instructions below to install the last stable version of Coq and
 additional packages. The instructions target an opam newcomer.</p>
 
+<p>Note that these instructions will also work for Opam 2.0 but this may require you to manually install any external dependencies. Opam will inform you what these are if they are required.</p>
+
+<p>For some operating systems, Opam (>=2.1) might still be unable to install external dependencies, which will mean you have to install them yourself.</p>
+
 <h2 id="platform">The Coq platform scripts</h2>
 
 <p>The <a href="https://github.com/coq/platform">Coq platform</a>
 provides interactive scripts that allow installing Coq and a standard
 set of packages through opam without having to learn anything about
-opam.  We recommend that you use
+  opam.</p>
+
+<p>Especially if you are a first-time user, we recommend that you use
```

Comment:
I'm not convinced that we should make it sound like we restrict this recommendation to first-time users. Maybe something like
```suggestion
<p>If a standard setup works for you, then we recommend that you use
```

Note that using the platform scripts creates a new switch which can later be modified by adding packages or even upgrading / downgrading them. This is to be contrasted with using prebuilt packages like the Snap package, the macOS or the Windows installers, which do not provide this flexibility.

Reply:
Btw, I've addressed this but I haven't added any information contrasting it with the Snap package. I also don't think this is the correct place to do this since this page only focuses on Opam. Contrasting with Snap should come in an earlier page where different installation methods are explained.

Reply:
Yes, my comment wasn't meant to be included in the documentation. This comparison is already done at https://coq.inria.fr/download:

>Beginners are encouraged to use one of the binary installers: we provide binary installers for Windows and macOS and a Snap package compatible with many Linux distributions.
>
>Experienced users are advised to run the scripts provided by the Coq platform to install from sources as this will allow them to install additional packages with opam.